packages/vite/LICENSE.md

```diff
@@ -1385,6 +1385,21 @@ Repository: git+https://github.com/mcollina/fastq.git
 
 ---------------------------------------
 
+## fdir
```

Comment:
Should fast-glob be removed? Or does that depend on my PR to the rollup plugins?

Reply:
It's still bundled because the dynamic import plugin depends on it. If your PR doesn't get merged before v6 release, I think we can patch it since we only use the `dynamicImportToGlob` function.